model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/velocity/velocity_advection.py

```diff
@@ -479,6 +657,14 @@ def run_corrector_step(
             offset_provider=self.grid.offset_providers,
         )
 
+        """
+        z_w_con_c (0:nlev):
```

Comment:
```suggestion
        z_w_con_c:
```
You don't need to specify bounds unless there is a very particular corner case